home-manager/services/code-syncer/sync.sh

```
@@ -277,6 +280,67 @@ install_extensions() {
   fi
 }
 
+# Special function to sync ALL extensions to VS Code Insiders without any filtering
+install_all_extensions_insiders() {
+  local target_name="code-insiders"
+  local cli_cmd=$(resolve_cli "$target_name")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)